apps/test/unit/gamelab/animationListModuleTest.js

```diff
@@ -82,4 +86,154 @@ describe('animationListModule', function () {
       expect(store.getState().animationList.propsByKey[key].sourceSize).not.to.be.undefined;
     });
   });
+
+  describe('action: set initial animationList', function () {
+    let oldWindowDashboard, server;
+    beforeEach(function () {
+      oldWindowDashboard = window.dashboard;
+      window.dashboard = {
+        project: {
+          getCurrentId() {return '';}
+        }
+      };
+
+      server = sinon.fakeServer.create();
+      server.respondWith('imageBody');
+    });
+
+    afterEach(function () {
+      server.restore();
+      window.dashboard = oldWindowDashboard;
+    });
+
+    it('when animationList has 1 item, selectedAnimation should be the animation', function () {
+      const key0 = 'anim0';
+      let animationList = {
```

Comment:
At one point weren't you writing a test to verify that for an animationList with more than one item, the first item would be selected by default?  You could extract a helper that builds a valid animationList of _n_ items and use it to write that test, too.